### PR TITLE
Starting guides for the publishing and querying

### PR DESCRIPTION
**Pull request by jmg-duarte**

### CODE REVIEW

File: web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+### Running the query
+
+Now that you're up to speed, the command we will be running is the following:
+
+<Tabs>
+<Windows>
+
+```
+.\ax events query localhost "FROM allEvents"
```
Comment (rkuhn) [inline web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx:62]: It wouldn’t make it significantly more difficult to add `--tag a` when publishing the first event and querying `FROM 'a'` here, and I’d think it yields are more empowering experience.

File: web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+1/z4tg-0@2023-08-08T15:57:34.763995+01:00: {"streamName": "discovery", "streamNr": 1}
+2/z4tg-0@2023-08-08T15:57:34.766578+01:00: {"streamName": "metrics", "streamNr": 2}
+3/z4tg-0@2023-08-08T15:57:34.769054+01:00: {"streamName": "files", "streamNr": 3}
+4/z4tg-0@2023-08-08T15:57:34.772122+01:00: {"NewListenAddr": ["12D3KooWSokZosGoYhKa5GxVjuDFoY2fPh7SLF6TcTS3bwGZhRzs", "/ip4/192.168.0.104/tcp/4001"]}
+5/z4tg-0@2023-08-08T15:57:34.774454+01:00: {"PinAdded": {"app_id": "com.actyx", "query": "FEATURES(zøg aggregate timeRange) FROM (((isLocal & appId(com.actyx)) & \'files:created\') & TIME >= 2023-08-08T02:57:34Z) SELECT _.cid END", "duration": {"secs": 604800, "nanos": 0}}}
+6/z4tg-0@2023-08-08T15:57:43.737290+01:00: "hello"
+```
+
+:::info
+You may have more events in your output, don't worry, that is completely normal!
```
Comment (rkuhn) [inline web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx:97]: When switching to tag `'a'`, this note should only say that with `FROM allEvents` you can also see all the Actyx internal events if you want to.

File: web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+:::
+
+If you notice on the last line, it's our payload!
+
+## Querying Actyx using the Node Manager
+
+The Node Manager is the GUI interface for Actyx, it's like the CLI's fancy cousin, the main difference being that you cannot publish events from the Node Manager.
```
Comment (rkuhn) [inline web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx:104]: Is the change for this line in a different PR?

File: web/developer.actyx.com/docs/tutorials/getting-started/machine-runner/state-payloads.mdx [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+```bash
+$ npm run start-robot
+
+> tomato-robot@1.0.0 start-robot
+> tsc && node src/index.js
+
+{
+  is: [Function: is],
+  as: [Function: as],
+  cast: [Function: cast],
```
Comment (rkuhn) [inline web/developer.actyx.com/docs/tutorials/getting-started/machine-runner/state-payloads.mdx:289]: To me dumping the whole JS object doesn’t add much value — how about removing `{ is, as, cast }` in the code above, noting that these are functions?

**Review by rkuhn — APPROVED** [review]
nice example!

File: web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx (hunk 3 above)
Comment (rkuhn) [reply web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx:104]: I meant that event publishing is now implemented in Node Manager — unless I’m mixing up multiverses.

Comment (jmg-duarte) [reply web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx:104]: Ah it is, but this tutorial predates that by weeks. If that's the case, I'm going to merge this as is and we'll add the publish for node manager on top of this

**Pull request merged**